runtime/descriptor.go

```diff
@@ -98,11 +98,13 @@ func propertySet(f *Frame, desc, inst, value *Object) *BaseException {
 
 // makeStructFieldDescriptor creates a descriptor with a getter that returns
 // the field given by fieldName from t's basis structure.
-func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object {
+func makeStructFieldDescriptor(t *Type, fieldName, propertyName string, writtable bool) *Object {
```

Comment:
You mean, an ``int`` from an ``enum`` instead of the ``writtable bool``? Well, this could accommodate more modes in the future if needed. Just confirm please.